README.md

```diff
@@ -93,6 +95,22 @@ In your view/page/dialog widget's State class, create an instance of FlutterSoun
 FlutterSound flutterSound = new FlutterSound();
 ```
 
+#### Initialize the player.
+In order to be able to execute all the player methods, you must initialize the player by calling the ```initialize``` method.
+You could also pass two functions as arguments of ```initialize```, and they will be triggered when the user tries to skip forward or backward using the buttons in the notification.
+Furthermore, your application should display media player controls only when the initialization finished successfully.
+
+```dart
+@override
+void initState() {
+	super.initState();
+	flutterSound.initialize()
+	.then((_) {
+          displayMediaPlayerControls();
```

Comment:
Just the question! So if the user doesn't need it we can remove the above code and use it as it was previously?

Reply:
The user must call ```initialize``` in order to use the audio player. Therefore this snipped cannot be removed.